src/main/java/edu/harvard/iq/dataverse/util/MailUtil.java

```diff
@@ -64,6 +64,8 @@ public static String getSubjectTextBasedOnNotification(UserNotification userNoti
                 return BundleUtil.getStringFromBundle("notification.email.import.filesystem.subject", rootDvNameAsList);
             case CHECKSUMIMPORT:
                 return BundleUtil.getStringFromBundle("notification.email.import.checksum.subject", rootDvNameAsList);
+            case CONFIRMEMAIL:
+                return BundleUtil.getStringFromBundle("notification.email.verifyEmail.subject", rootDvNameAsList);
```

Comment:
As I just mentioned to @sekmiller some day it might be nice to associate these bundle keys with the UserNotification object. Maybe you just call userNotification.getEmailSubject() or something. Out of scope for now but something to consider for the future.